src/models/node.js

```diff
@@ -1329,6 +1321,40 @@ const Node = {
   },
 
   /**
+   * Split a node by `path` after 'count' children.
+   * Does not work on Text nodes. Use `Node.splitNode` to split text nodes as well.
+   *
+   * @param {Array} path
+   * @param {Number} count
+   * @return {Node}
+   */
+
+  splitNodeAfter(path, count) {
```

Comment:
We need to have a way to simply join two nodes. So instead, we could keep `joinNode` that way, and have a `mergeNode` that does what you say. We would have 4 operations, 2 of them being the reverse of the two others:

```
splitNode <-> mergeNode
splitNodeAfter <-> joinNode
```

The names could be better